testlog/testlog_test.go

```diff
@@ -0,0 +1,31 @@
+package testlog
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/getlantern/golog"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	expectedCapture = `ERROR mytest: testlog_test.go:24 error 1
+DEBUG mytest: buffer.go:54 debug 1
```

Comment:
This is a weird effect of writing to a buffer, not sure why it's happening.

Reply:
Yeah I noticed it somewhere as well. I bet it relates to how we call `runtime.Callers` and/or `runtime.FuncForPC` - just got the wrong pointer on the call stack.